Cargo.toml

```diff
@@ -16,10 +16,12 @@ members = [
 ]
 
 exclude = [
+  # temporarily exclude `kani-verifier` until we don't have colliding binary names (cargo-kani)
+  "src/kani-verifier",
```

Comment:
Since we will no longer publish the other crates to crates.io, we probably want to move them out of src and only keep this new crate in.

This will also fix the readme issue.

Reply:
I don't understand this comment. We possibly need to chat about this because I'm not sure how best to solve this issue, and this sounds like you understand something here that I don't yet.